tests/ui_json_test.py

```diff
@@ -583,6 +596,20 @@ def test_input_file(tmp_path):
             raise ValueError(f"Input '{key}' differs from the output.")
 
 
+def test_write_ui_json(tmp_path):
+    # Make sure that none2str is applied in dict_mapper
+    workspace = get_workspace(tmp_path)
+    ui_json = deepcopy(default_ui_json)
+    ui_json["geoh5"] = workspace
+    ui_json["test"] = templates.float_parameter(optional="disabled")
```

Comment:
Can we change this whole business of "enabled" "disabled" to True or False instead? Just need to update the `optional_parameter` method in `templates`